<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IToolExport</class>
 <widget class="QWidget" name="IToolExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelHelp">
        <property name="text">
         <string>do not translate</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CItemTreeWidget" name="itemTree" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="labelNote">
        <property name="text">
         <string>do not translate</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboExport">
        <item>
         <property name="text">
          <string>Garmin BirdsEye (*.jnx)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TwoNav Raster (*.rmap)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="CToolExportJnx" name="pageBirdsEyeJnx"/>
        <widget class="QWidget" name="pageTwoNavRmap">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>not implemented yet</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineTragetFile"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/32x32/Apply.png</normaloff>:/icons/32x32/Apply.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushCancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icons/32x32/Cancel.png</normaloff>:/icons/32x32/Cancel.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNoMap2jnx">
     <property name="text">
      <string>&lt;b style='color: red;'&gt;No &quot;qmt_map2jnx&quot; found. Please check setup!&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionTargetFilename">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/PathBlue.png</normaloff>:/icons/32x32/PathBlue.png</iconset>
   </property>
   <property name="text">
    <string>Target Filename</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CToolExportJnx</class>
   <extends>QWidget</extends>
   <header>tool/export/CToolExportJnx.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CItemTreeWidget</class>
   <extends>QWidget</extends>
   <header>items/CItemTreeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
